plugins/fabrik_element/googlemap/googlemap.js

```diff
@@ -212,6 +212,7 @@ define(['jquery', 'fab/element', 'lib/debounce/jquery.ba-throttle-debounce', 'fa
             if (!this.options.staticmap) {
 
                 var zoomControlStyle = this.options.control === 'GSmallMapControl' ? google.maps.ZoomControlStyle.SMALL : google.maps.ZoomControlStyle.LARGE;
+				var vzoomControl = this.options.control !== 'none';
```

Comment:
Would it be better to do `var vzoomControl = this.options.control !== '';` in order that existing selections of 'none' actually do what it says on the can?

Yes - this would break backward compatibility, but if user has selected none and it still shows it, then that seems to me to be a bug rather than a feature.

Reply:
As nobody can currently save the "None" option, and even if they had (prior to some change in JForms a while back) it wouldn't have been working, requiring people to re-save the map element so the value becomes 'none' is the only fix.

Reply:
Ah - I had not realised that you could not save with it set to "None".